### PR TITLE
[VEN-2810][VEN-2811][VEN-2812]: VIP for VTreasury, ACM and Oracle on Opmainnet

### PR DESCRIPTION
**Pull request by Debugger022**
## Description

Resolves `VEN-2810`, `VEN-2811`, `VEN-2812`

### CODE REVIEW

File: src/networkAddresses.ts [hunk 1]
```diff
@@ -199,4 +199,14 @@ export const NETWORK_ADDRESSES = {
     CHAINLINK_ORACLE: "0x493C3f543AEa37EefF17D823f27Cb1feAB9f3143",
     POOL_REGISTRY: "0x6538C861C7A6997602311342657b9143dD9E8152",
   },
+  opmainnet: {
+    NORMAL_TIMELOCK: "", // To be deployed
+    VTREASURY: "0x104c01EB7b4664551BE6A9bdB26a8C5c6Be7d3da",
+    GUARDIAN: "0x2e94dd14E81999CdBF5deDE31938beD7308354b3",
+    RESILIENT_ORACLE: "0x21FC48569bd3a6623281f55FC1F8B48B9386907b",
+    CHAINLINK_ORACLE: "0x1076e5A60F1aC98e6f361813138275F1179BEb52",
+    BOUND_VALIDATOR: "0x37A04a1eF784448377a19F2b1b67cD40c09eA505",
```
Comment (chechu) [inline src/networkAddresses.ts:208]: We don't use/need the bound validator contract in the framework. I would remove it from here and I would define it in the multisig TX where it's used

File: multisig/simulations/opmainnet/vip-000/index.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+      expect(await chainLinkOracle.owner()).to.equal(opmainnet.GUARDIAN);
+      expect(await boundValidator.owner()).to.equal(opmainnet.GUARDIAN);
+    });
+
+    it("validate asset prices", async () => {
+      for (let i = 0; i < assetConfigs.length; i++) {
+        const assetConfig = assetConfigs[i];
+        const price = await resilientOracle.getPrice(assetConfig.address);
+
+        console.log("-------------price assetConfig", assetConfig.name, " ", price);
```
Comment (chechu) [inline multisig/simulations/opmainnet/vip-000/index.ts:120]: I would remove this log

Comment (Debugger022) [reply multisig/simulations/opmainnet/vip-000/index.ts]: fixed, https://github.com/VenusProtocol/vips/pull/386/commits/0e3d5904e56f8af9f8844016daa585a02c256a51

File: src/networkAddresses.ts (hunk 1 above)
Comment (Debugger022) [reply src/networkAddresses.ts]: fixed, https://github.com/VenusProtocol/vips/pull/386/commits/0e3d5904e56f8af9f8844016daa585a02c256a51

File: src/multisig/utils.ts [hunk 3]
```diff
@@ -122,6 +122,19 @@ export const getContractNetworks = (chainId: number): ContractNetworksConfig =>
         simulateTxAccessorAddress: "0xe7A7552B0Cff45E837e1422b479d2dEDE748f571",
       },
     },
+    // safe v1.4.1
+    opmainnet: {
```
Comment (GitGuru7) [inline src/multisig/utils.ts:126]: Only testnet addresses are stored here. I think we should remove it.

Comment (Debugger022) [reply src/multisig/utils.ts]: fixed, https://github.com/VenusProtocol/vips/pull/386/commits/213713fb4032b9dd9b7e580c0ec8d183a5a3ef8b

**Review by chechu — APPROVED** [review]

**Review by GitGuru7 — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/368?chainId=56

**Review by chechu — APPROVED** [review]

**Pull request merged**